test/argmax.jl

```diff
@@ -108,8 +112,8 @@ data_train, data_test = generate_dataset(
 
 ## Test loop
 
-for pipeline in pipelines_imitation_θ
-    pipeline_1 = deepcopy(pipeline)
+for k in eachindex(pipelines_imitation_θ)
```

Comment:
Why do we need the index here ?

Reply:
It was a remainder of my attempt at multithreading tests

Reply:
https://discourse.julialang.org/t/multi-threaded-testing/95372